CHANGELOG.md

```diff
@@ -3,3 +3,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.0.0] - 2018-08-28
+### Changed
+- Changed waggledance version from 2.3.1 to 2.4.2. See [#7](https://github.com/ExpediaInc/apiary-waggledance-docker/issues/7).
+
+[Unreleased]: https://github.com/ExpediaInc/apiary-waggledance-docker/compare/v1.0.0...HEAD
```

Comment:
We normally don't bother with the git commit tags as it's just one more thing to maintain but perhaps it makes more sense for these kinds of projects that don't produce separate artifacts?

Reply:
@massdosage I guess I'm confused. This project produces a docker image, won't we tag the docker image?